### PR TITLE
MULE-14871: Update mule-credentials-vault documentation using secure properties module

### PR DESCRIPTION
**Pull request by mbuchwald**

### CODE REVIEW

File: runtime-manager/v/latest/secure-application-properties.adoc [hunk 1]
```diff
@@ -197,7 +197,7 @@ For example, if a production application needs to update the database user and p
 The previous discussion indicates that for Mule applications that will only be deployed into CloudHub workers, you may not need to encrypt properties, but instead just flag those properties as safely hidden propreties in the Mule application's 'mule-app.properties' file's `secure.properties` entry. 
 
 == See Also
-* link:/mule-user-guide/v/4.1/mule-credentials-vault[Credentials Vault]
+* link:/mule-user-guide/v/4.1/secure-configuration-properties.adoc[Secure Configuration Properties]
```
Comment (aiannucci) [inline runtime-manager/v/latest/secure-application-properties.adoc]: Remove .adoc extension

File: runtime-manager/v/latest/secure-application-properties.adoc [hunk 2]
```diff
@@ -111,7 +111,7 @@ When moving applications between *sandboxes*, safely hidden application property
 == Encrypted Application Properties vs. Safely Hidden Application Properties
 Mule applications can also store properties with encrypted values using secure properties placeholders.
 
-See link:/mule-user-guide/v/4.1/mule-credentials-vault[Credentials Vault] for more information on creating and using secure properties placeholders. 
+See link:/mule-user-guide/v/4.1/secure-configuration-properties.adoc[Secure Configuration Properties] for more information on creating and using secure configuration properties.
```
Comment (aiannucci) [inline runtime-manager/v/latest/secure-application-properties.adoc]: Remove .adoc extension

File: runtime-manager/v/latest/secure-application-properties.adoc [hunk 3]
```diff
@@ -15,7 +15,7 @@ Internally, safely hidden application properties are encrypted and stored in a M
 
 Safely hidden application properties are a feature that is currently only available for setting via the Runtime Manager UI for applications deployed to CloudHub workers. For applications that you deploy through any other link:/runtime-manager/deployment-strategies[deployment strategy], you need to use secure property placeholder files bundled with your application zip file.
 
-Safely hidden application properties are a separate feature and concept from encryted Mule application properties that are stored in secure properties placeholder files, and securely access through a link:/mule-user-guide/v/3.9/mule-credentials-vault[Credentials Vault]. 
+Safely hidden application properties are a separate feature and concept from encryted Mule application properties that are stored in link:/mule-user-guide/v/4.1/secure-configuration-properties.adoc[Secure Configuration Properties] files.
```
Comment (aiannucci) [inline runtime-manager/v/latest/secure-application-properties.adoc]: Remove .adoc extension

File: mule-user-guide/v/4.1/4.0_additions/configuring-properties.adoc [hunk 4]
```diff
@@ -204,11 +167,16 @@ To create an environment variable or application property:
...
-== See Also
+== Properties Hierarchy
 
-* Read more about mule messages and their properties in our link:http://blogs.mulesoft.com/dev/anypoint-platform-dev/mule-school-the-mulemessage-property-scopes-and-variables/[Mule School: The MuleMessage, Property Scopes, and Variables].
-* link:/mule-user-guide/v/3.9/deploying-to-multiple-environments[Deploying to Multiple Environments]
+Configuration properties can be overwritten. The Hierarchy in which these are treated is: 
+1. Environment Properties
+1. System Properties
+1. Deployment Properties
+1. Application Properties
```
Comment (aiannucci) [inline mule-user-guide/v/4.1/4.0_additions/configuring-properties.adoc]: Wrong numbering

File: mule-user-guide/v/4.1/4.0_additions/configuring-properties.adoc [hunk 5]
```diff
@@ -28,57 +28,42 @@ You can use the `<global-property>` element to set a placeholder value from with
 
 [WARNING]
 ====
-If you deploy multiple applications through a link:/mule-user-guide/v/3.9/shared-resources[Shared Resources] structure, don't set anything in the properties files, as there might potentially be conflicts between the various apps that share a domain. Instead, set environment variables over the scope of the deployed app, its domain, and other apps under that domain.
+If you deploy multiple applications through a link:/mule-user-guide/v/4.1/shared-resources[Shared Resources] structure, don't set anything in the properties files, as there might potentially be conflicts between the various apps that share a domain. Instead, set environment variables over the scope of the deployed app, its domain, and other apps under that domain.
```
Comment (pablolagreca) [inline mule-user-guide/v/4.1/4.0_additions/configuring-properties.adoc]: This is wrong. Application defined properties have precedence over domain defined properties and it's perfectly fine to use them in Mule 4.

File: mule-user-guide/v/4.1/4.0_additions/configuring-properties.adoc [hunk 6]
```diff
@@ -16,7 +16,7 @@ The values for these placeholders can be made available in a variety of ways, as
 
 == Global Properties
 
-You can use the `<global-property>` element to set a placeholder value from within your Mule configuration, such as from within another Mule configuration file. Do not use the global property syntax to reference the values from a `.properties` file. To reference properties from a file, read the section on properties files.
+You can use the `<global-property>` element to set a placeholder value from within your Mule configuration, such as from within another Mule configuration file. Do not use the global property syntax to reference the values from a `.yaml` or `.properties` file. To reference properties from a file, read the section on properties files.
```
Comment (pablolagreca) [inline mule-user-guide/v/4.1/4.0_additions/configuring-properties.adoc]: this is wrong. Please double check but is perfectly fine to reference a property defined in a properties file from a global-property.

File: mule-user-guide/v/4.1/4.0_additions/configuring-properties.adoc [hunk 7]
```diff
@@ -193,7 +156,7 @@ image:Environment+Tab2.png[Environment+Tab2]
 
 If you deploy your application to link:/runtime-manager/[Runtime Manager], you can also set properties through the Runtime Manager console. These can be defined when link:/runtime-manager/deploying-to-cloudhub[Deploying to CloudHub], or on an link:/runtime-manager/managing-applications-on-cloudhub[already running application].
 
-*Note:* If you also have the same variables set in the `mule-app.properties` file inside your application, the environment variables you set here in the console always override the values in 'mule-app.properties' when your application deploys.
+*Note:* If you also have the same variables set in the `mule-app.yaml` file inside your application, the environment variables you set here in the console always override the values in 'mule-app.yaml' when your application deploys.
```
Comment (pablolagreca) [inline mule-user-guide/v/4.1/4.0_additions/configuring-properties.adoc]: Remove this note. It made sense for mule-app.properties but not anymore

File: mule-user-guide/v/4.1/4.0_additions/configuring-properties.adoc (hunk 6 above)
Comment (mbuchwald) [reply mule-user-guide/v/4.1/4.0_additions/configuring-properties.adoc]: We even have tests that prove this wrong...

**Comment by dukesphere** [comment]
manually "merged" these into new mule4-user-guide folder and the runtime-manager folder

**Pull request closed without merging**